src/to-have-display-value.js

```diff
@@ -0,0 +1,34 @@
+import {matcherHint} from 'jest-matcher-utils'
+
+import {checkHtmlElement, getMessage} from './utils'
+
+export function toHaveDisplayValue(htmlElement, expectedValue) {
+  checkHtmlElement(htmlElement, toHaveDisplayValue, this)
+
+  if (htmlElement.tagName.toLowerCase() !== 'select') {
+    throw new Error(
+      '.toHaveDisplayValue() currently supports select elements only, try to use .toHaveValue() or toBeChecked() instead.',
```

Comment:
Added dot to `toBeChecked`
```suggestion
      '.toHaveDisplayValue() currently supports select elements only, try to use .toHaveValue() or .toBeChecked() instead.',
```